src/test/java/com/elixr/springbootfile/controller/ControllerTest.java

```diff
@@ -0,0 +1,101 @@
+package com.elixr.springbootfile.controller;
+
+import com.elixr.springbootfile.service.FileService;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMultipartHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class ControllerTest {
+
+    @MockBean
+    private FileService fileService;
+
+    @Autowired
+    private Controller controller;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.openMocks(this);
+        this.mockMvc = MockMvcBuilders.standaloneSetup(controller).build();
+    }
+
+    @Test
+    void createFile() throws Exception {
+        MockMultipartFile sampleFile = new MockMultipartFile("file",
+                "some.txt",
+                "text/plain",
+                "This is the file content".getBytes());
+        MockMultipartHttpServletRequestBuilder multipartRequest =
+                MockMvcRequestBuilders.multipart("/upload");
+        mockMvc.perform(multipartRequest.file(sampleFile).param("userName", "bikas"))
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    public void testUploadFileNoFileProvided() throws Exception {
```

Comment:
You are not providing the file or the userName, so test name should reflect that. testUploadNoParametersProvided. You should also have tests for the below conditions:
just the file missing 
just the userName missing
file is not a txt file

Please inform Vikas and Abhijit as well

Reply:
ok will change it to testUploadNoParametersProvided and will work for the above mentioned test cases.
I have informed Vikas and Abhijit to add those test cases.